scipy/stats/stats.py

```diff
@@ -1015,16 +1015,18 @@ def tsem(a, limits=None, inclusive=(True, True), axis=0, ddof=1):
 
 def moment(a, moment=1, axis=0):
     """
-    Calculates the nth moment about the mean for a sample.
+    Calculates the nth central moment about the mean for a sample.
+
```

Comment:
Central moment is by definition the moment about the mean. I'd only leave on of qualifiers.